modules/custom_role_iam/variables.tf

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Base resource supports stage argument, is there a reason we are not supporting this?
https://www.terraform.io/docs/providers/google/r/google_project_iam_custom_role.html#stage

Reply:
I am not sure how a user would use stage argument while creating a custom role. Do you think it should be included?

Reply:
On second thought, I added stage with a default of GA as mentioned on the page.